felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Ressurs.kt

```diff
@@ -29,49 +29,48 @@ data class Ressurs<T>(
         FUNKSJONELL_FEIL
     }
 
+    init {
+        if (status == Status.SUKSESS)
+        requireNotNull(data){"Hvis status er suksess kan ikke data være null."}
```

Comment:
Er det greit med empty?

Reply:
Usikker. Motivasjonen bak endringen er alle steder vi tester for om suksess inneholder null, og kaster feil.

Reply:
Er vel ikke så lett å sjekke for not empty på <T : Any>